azext_iot/_params.py

```diff
@@ -879,6 +892,18 @@ def load_arguments(self, _):
             options_list=["--auth-type"],
             arg_type=hub_auth_type_dataplane_param_type,
         )
+        context.argument(
+            "custom_metric_queries",
+            nargs="+",
+            options_list=["--custom-metric-queries", "--cmq"],
+            help="An alternative way of input style(narg key-value pair) to --metrics and intended to replace it in the future.",
```

Comment:
We should consider adding in the existing `--metrics` description that `--custom-metrics-queries` is recommended.

Reply:
Same for the `--labels` description and recommending `--custom-labels`.

Reply:
I changed to "IoT Edge deployment metric definitions. Provide file path or raw json. Using --custom-metric-queries instead of --metrics is recommended." does this sound good?